pkg/commands/process/settings/rules/javascript/third_parties/sentry.yml

```
@@ -0,0 +1,72 @@
+patterns:
+  - pattern: |
+      Sentry.addBreadcrumb($<DATA_TYPE>)
+    filters:
+      - variable: DATA_TYPE
+        detection: datatype
```

Can we use a VAR here or is it not supported for JS e.g. 
```suggestion
  - pattern: |
      Sentry.$<VAR>($<DATA_TYPE>)
    filters:
      - variable: DATA_TYPE
        detection: datatype
      - variable: VAR
         values: 
           - addBreadcrumb
           - captureMessage
           ...
```

---

We probably can but I keep that for after :)